.dockerignore

```diff
@@ -3,11 +3,10 @@ node_modules
 .dockerignore
 .editorconfig
 .env
-.eslintignore
-.eslintrc
```

Comment:
want these to be included in the e2e_runner container so lab can lint e2e test files